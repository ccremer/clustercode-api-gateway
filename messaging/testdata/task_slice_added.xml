<?xml version="1.0" encoding="UTF-8"?>
<slice_added_event version="1">
  <job_id>620b8251-52a1-4ecd-8adc-4fb280214bba</job_id>
  <priority>1</priority>
  <slice_nr>34</slice_nr>
  <file>clustercode://base_dir:0/asdf.xml</file>
  <args>
    <arg>arg1</arg>
    <arg>arg with space</arg>
  </args>
  <std_streams>
    <stream>
      <fd>2</fd>
      <l>aösldkafjsöldkjf</l>
    </stream>
    <stream>
      <fd>2</fd>
      <l>aösldkafjsöldkjf</l>
    </stream>
  </std_streams>
</slice_added_event>
