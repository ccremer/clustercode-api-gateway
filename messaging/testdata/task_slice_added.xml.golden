<slice_added_event>
    <job_id>620b8251-52a1-4ecd-8adc-4fb280214bba</job_id>
    <priority>1</priority>
    <args>
        <arg>maybe</arg>
    </args>
    <std_streams>
        <stream>
            <fd>2</fd>
            <l>aösldkafjsöldkjf</l>
        </stream>
    </std_streams>
</slice_added_event>
